### DISCUSSION TITLE
Add new input with counter

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#531] [Proposals]
### Context
If you think there is a component missing in Mística and you feel it is worthy to create it globally and therefore any team could use it, please follow the process below so we can integrate this new component from Design to Development.

**Have you read the [Contributing Guidelines on Mística](https://brandfactory.telefonica.com/document/1846#/contribute/how-to-contribute-1)?**
- [x] Check to confirm

**When will I receive a reply?**
We usually review component request Tuesdays at 16:00 (CEST), after the meeting, we will reply in this component request.

---

### Is your component request related to a problem? Please describe.
Doesn't exists an input with + - symbol to count amounts

### Describe the solution you'd like and needs
Used in shop
![image](https://user-images.githubusercontent.com/6722153/154369641-c8a67c84-1ed9-42b4-b4f4-98bb8075f3f2.png)

### Additional context
Ask to other teams like COL that have this component

**Comment by aweell** [comment]
I close this discussion since is already covered in #1364
